src/main.rs

```diff
@@ -58,6 +58,12 @@ fn app_args<'a>() -> clap::ArgMatches<'a> {
         .long("select-fg-color")
         .default_value("blue"),
     )
+    .arg(
+      Arg::with_name("select_background_color")
+        .help("Sets the foregroud color for selection")
```

Comment:
_Sets the backgroud color for selection_

BTW: Could you fix the typo in "foregroud" word, please? 🙏

Reply:
Done 👍